packages/Button/stories/examples/ButtonStates.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { Story } from "storybook/assets/styles/common.styles";
+import PlusIcon from "@paprika/icon/lib/Add";
+import Button from "../../src";
+
+const ExampleStory = () => {
+  return (
+    <Story>
+      <Button kind="flat">Flat</Button>
+      <Button kind="secondary">Secondary</Button>
+      <Button kind="destructive">Destructive</Button>
+      <Button isActive>Active</Button>
+      <Button isDisabled>Disabled</Button>
+      <Button isPending>Pending</Button>
+      <Button isDropdown>Dropdown</Button>
+      <Button icon={<PlusIcon />} />
```

Comment:
You should use `<Button.Icon>` for this example.  The "Basic" story has examples of how it's used, but essentially it's: `<Button.Icon><Icon /></Button.Icon>`

Reply:
@mikrotron is `<Button icon={<PlusIcon />}>Icon</Button>` okay though? I see that no text shows up for `<Button.Icon><PlusIcon />Icon</Button.Icon>`.